mewarpx/mewarpx/mwxrun.py

```diff
@@ -183,16 +183,16 @@ def get_npart_species_dict(self):
 
         return npart_dict
 
-    def get_rho_grid(self):
+    def get_rho_grid(self, include_ghosts=True):
         """Get rho segments on the grid for each tile of each processor.
 
         Returns:
             A list of numpy arrays, the list has an array for every tile and
             each array has dimensions given by the number of cells in that tile.
         """
-        return _libwarpx.get_mesh_charge_density_fp(self.lev)
+        return _libwarpx._get_mesh_field_list(_libwarpx.libwarpx.warpx_getChargeDensityFP, self.lev, None, include_ghosts)
```

Comment:
Do we actually use this at all? If not we should just remove it. If we find we ever do want it we can put it back then.